s-ramp-ide/srampRepositoryBrowser/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,48 @@
+Manifest-Version: 1.0
```

Comment:
I searched for some solution how to generate MANIFEST.MF and build.properties files with using tycho. By eclipse-wiki it must be two separate projects. First must be clean maven project (without tycho). In this project there are generated this files. Second project is tycho project. This project uses project builded in first step and packed this project into eclipse-plugin. I thing it is better commit this two files.